CONTRIBUTING.md

```diff
@@ -66,75 +66,60 @@ Then every time you want to test a new version, you can just:
+    value: clusterlogforwarders.v1.logging.openshift.io
+  - name: userWorkloadTracesCollection
+    value: opentelemetrycollectors.v1alpha1.opentelemetry.io
+  - name: userWorkloadInstrumentation
+    value: instrumentations.v1alpha1.opentelemetry.io
 ``` 
 
-Supported keys are `metricsDisabled`, `loggingDisabled` and `tracingDisabled`
+Supported keys are:
+- `platformLogsCollection`: Supports values `clusterlogforwarders.v1.logging.openshift.io`
+- `userWorkloadLogsCollection`: Supports values `clusterlogforwarders.v1.logging.openshift.io`
+- `userWorkloadTracesCollection`: Supports values `opentelemetrycollectors.v1alpha1.opentelemetry.io`
+- `userWorkloadTracesInstrumentation`: Supports values `instrumentations.v1alpha1.opentelemetry.io`
 
-## Install the addon on a Spoke Cluster
+__Note__: Keys can hold multiple values separated by semicolon, e.g. `clusterlogforwarders.v1.logging.openshift.io;opentelemetrycollectors.v1alpha1.opentelemetry.io`.
 
-To actually install the addon on a spoke cluster, you need to:
+### Configuring Platform Observability Capabilities
 
-1. Have the addon manager running on the hub cluster.
-2. Create the necessary Kubernetes resources in the namespace of the spoke
-    cluster that will be used by the addon to generate the `ManifestWorks`, e.g.,
-    `secrets`, `configmaps`.
-3. Create the `ManagedClusterAddon` resource in the namespace of the spoke
-    cluster.
+#### Logs Collection
 
-```yaml
-apiVersion: addon.open-cluster-management.io/v1alpha1
-kind: ManagedClusterAddOn
-metadata:
-  name: multicluster-observability-addon
-  namespace: spoke-1
-spec:
-  installNamespace: open-cluster-management-agent-addon
-  configs:
-  - resource: configmaps
-    name: spoke-1
-    namespace: spoke-1
-  - resource: secrets
-    name: spoke-1
-    namespace: spoke-1
-```
-
-4. Once a `ManagedClusterAddon` is reconciled successfuly by the addon we can
-   look for the `ManifestWorks`
-
-```shell
-oc -n spoke-1 get manifestworks addon-multicluster-observability-addon-deploy-0
-```
+Currently the addon supports configuration to send logs either to:
 
-### Configuring Metrics
+- CloudWatch: requires the auth configmap to be specified
+- Loki: requires the auth configmap, the url configmap and optionally the inject ca configmap
 
-Currently the addon doesn't support any configuration, so no configuration is needed at the `ManagedClusterAddOn` level. However, the addon has a dependency with MCO. 
-Nowadays the addon supports the collection of metrics from the spoke clusters. These metrics are sent to an MCO instance running in the Hub.
+### Configuring User Workloads Observability Capabilities
 
-### Configuring Logs
+#### Logs Collection
 
 Currently the addon supports configuration to send logs either to:
 
 - CloudWatch: requires the auth configmap to be specified
 - Loki: requires the auth configmap, the url configmap and optionally the inject ca configmap
 
-### Configuring Traces
+### Traces Collection & Instrumentation 
```

Comment:
This should not be together. The Instrumentation CR injects libraries that provide metrics + traces and, in the future, I think they will include logs.

Reply:
Good point, I can expand the markdown into separate sections then.

Reply:
I am going to remove this `& Instrumentation` until you guys contribute reconciling this resource in the addon.